CONTRIBUTING.md

```diff
@@ -0,0 +1,291 @@
+# Contributing to Designsystemet
+
+First off, thanks for taking the time to contribute! ❤️
+
+All types of contributions are encouraged and valued. See the [Table of Contents](#table-of-contents) for different ways to help and details about how this project handles them. Please make sure to read the relevant section before making your contribution. It will make it a lot easier for us maintainers and smooth out the experience for all involved. The community looks forward to your contributions. 🎉
+
+> And if you like the project, but just don't have time to contribute, that's fine. There are other easy ways to support the project and show your appreciation, which we would also be very happy about:
+>
+> - Star the project
+> - Refer this project in your project's readme
+> - Mention the project at local meetups and tell your friends/colleagues
+
+## Table of Contents
+
+- [Code of Conduct](#code-of-conduct)
+- [I Have a Question](#i-have-a-question)
+- [I Want To Contribute](#i-want-to-contribute)
+  - [Reporting Bugs](#reporting-bugs)
+  - [Suggesting Enhancements](#suggesting-enhancements)
```

Comment:
Suggest we strip this down to just on chapter, which is getting in contact or creating an issue. 

Github already have a good templating issue for different _kinds_ of issue, so information thats relevant to the different types should be added there as to avoid duplicate docs.

Example of bug report:
![image](https://github.com/digdir/designsystem/assets/1070981/545d49e4-f92e-4e5d-ac79-a3df0e465ba4)

Reply:
Stripped this down a lot